tests/integration/java/com/github/bazel_contrib/target_determinator/integration/BUILD.bazel

```diff
@@ -120,3 +121,19 @@ java_library(
         artifact("org.hamcrest:hamcrest-all"),
     ],
 )
+
+copy_file(
+    name = "bazel-differ-binary",
+    src = ":original-bazel-differ",
+    out = "bazel-differ",
+)
+
+alias(
+    name = "original-bazel-differ",
+    actual = select({
+        "//:linux_arm64": "@bazel_differ_linux_arm64//file",
+        "//:linux_x86_64": "@bazel_differ_linux_x86_64//file",
+        "//:macos_arm64": "@bazel_differ_darwin_arm64//file",
+        "//:macos_x86_64": "@bazel_differ_darwin_x86_64//file",
```

Comment:
Please add support for windows as well as it's a supported platform (see e.g. https://github.com/bazel-contrib/target-determinator/blob/61d25f5e13a4b2205b338531b929464e3ed5eb3d/rules/multi_platform_go_binary.bzl#L8)

Reply:
While _we_ may support Windows, `bazel-differ` don't test with it, and they don't ship a binary for it, so it seems reasonable to assume that they don't support it.

Reply:
Oh right, we're cross-building the windows binary from linux 👍